### DISCUSSION TITLE
Show the Floccus Bookmarks web page

### DISCUSSION THREAD

**Discussion by Didier3L** [floccusaddon/floccus#1726] [Q&A]
Hello,

I synchronize my Bookmarks in webdav on my NAS
When I want to display the bookmarks.xbel file in my browser the display is unusable

![image](https://github.com/user-attachments/assets/d767094b-179a-49ef-bd0f-6fce74d11ff8)
Is there a way to be able to display it like the application?
![image](https://github.com/user-attachments/assets/f3b37123-f57a-4320-97f1-ca76545cb2ae)

**Comment by marcelklehr** [comment]
Hey @Didier3L 
You could switch to syncing to an HTML file. That's slightly more useful than the XML file. It'll give you links to click on and the hierarchy, but also no icons. The nice thing about the HTML format is also that you should be able to import the bookmarks in it into your browser directly, without floccus.
